packages/core/types/test/test.js

```diff
@@ -1,96 +0,0 @@
-"use strict";
```

Comment:
This is a .gitignored file that can be removed as it is generated when needed during testing.